app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -95,17 +95,27 @@ class FollowUsers extends Component {
   render() {
     const { users, selectedUsers } = this.state;
     const { prev } = this.props;
+    let followingStatus;
+    if (selectedUsers < 2) {
+      followingStatus = "You're not following anyone";
+    } else {
+      const everyone =
+        selectedUsers.length === users.length ? ' (everyone)' : '';
+      const object = selectedUsers.length === 1 ? 'person' : 'people';
+      followingStatus = `You're following ${selectedUsers.length} ${object}${everyone}`;
```

Comment:
I find this interpolation hard to follow 😕 What if we just split this out into an `else if` for following a certain amount of people, and then one more `else if` for following everyone?

Reply:
Great suggestion. Updated!